acceptance/ui/features/cli.feature

```diff
@@ -10,27 +10,49 @@ Feature: CLI Validation
      And only the default host is added
      And that the "table://applications/defaultApp/template" application with the "table://applications/defaultApp/id" Deployment ID is added
 
+  @port
   Scenario: List the port public endpoints
     Given that the "port0" port is added
-    Then I should see the port public endpoint "port0" in the service
+    Then I should see the port public endpoint "port0" in the list output
 
+  @port
   Scenario: Add a new port public endpoint
     Given that the "port1" port does not exist
       And that the "port1" port is added
     Then I should see the port public endpoint "port1" in the service
 
+  @port
   Scenario: Delete the port public endpoint
     Given that the "port0" port is added
       And the port public endpoint "port0" is removed
     Then I should not see the port public endpoint "port0" in the service
 
+  @port
   Scenario: Disable a port public endpoint
     Given that the "port0" port is added
       And that the port public endpoint "port0" is disabled
     Then the port public endpoint "port0" should be "disabled"
 
+  @port
   Scenario: Disable and enable a port public endpoint
     Given that the "port0" port is added
       And that the port public endpoint "port0" is disabled
       And that the port public endpoint "port0" is enabled
     Then the port public endpoint "port0" should be "enabled"
+
+  @vhost
+  Scenario: List the vhost public endpoints
```

Comment:
No need to do this now, but at some point we may want separate CLI feature files for different CLI features rather than all CLI calls in one file.